app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -230,9 +233,21 @@ export class ProfileForm extends Component {
                   label: 'Bio',
                   placeholder_text: 'Tell us a little about yourself',
                   required: false,
+                  maxLength: SUMMARY_MAXLENGTH,
                 }}
                 onFieldChange={this.handleFieldChange}
               />
+              <p
+                id="summary-description"
+                class="crayons-field__description align-right"
+              >
+                <span class="screen-reader-only" aria-live="polite">
```

Comment:
Love the use of `aria-live="polite"` within the `screen-ready-only` section here 👏🏼